src/org/labkey/snd/SNDManager.java

```diff
@@ -1702,19 +1703,31 @@ public void registerAttributeLookups(Container c, User u, String schema, String
      * Gets event data for a given event.  This includes the data from snd.EventData and the attribute data stored in
      * exp.ObjectProperty.  Recursively iterates through subpackages to get data.
      */
-    private EventData getEventData(Container c, User u, int eventDataId, SuperPackage superPackage, BatchValidationException errors)
+    private EventData getEventData(Container c, User u, @Nullable Integer eventDataId, @NotNull SuperPackage superPackage, BatchValidationException errors)
     {
         UserSchema schema = getSndUserSchema(c, u);
         TableInfo eventDataTable = getTableInfo(schema, SNDSchema.EVENTDATA_TABLE_NAME);
 
+        Map<String, ObjectProperty> properties = null;
+        EventData eventData = null;
+        TableSelector ts = null;
+
         // Get from EventData table
-        SimpleFilter filter = new SimpleFilter(FieldKey.fromParts("EventDataId"), eventDataId, CompareType.EQUAL);
-        TableSelector ts = new TableSelector(eventDataTable, filter, null);
+        if (eventDataId != null)
+        {
+            SimpleFilter filter = new SimpleFilter(FieldKey.fromParts("EventDataId"), eventDataId, CompareType.EQUAL);
+            ts = new TableSelector(eventDataTable, filter, null);
 
-        EventData eventData = ts.getObject(EventData.class);
+            eventData = ts.getObject(EventData.class);
 
-        OntologyManager.getProperties(c, eventData.getObjectURI());
-        Map<String, ObjectProperty> properties = OntologyManager.getPropertyObjects(c, eventData.getObjectURI());
+            OntologyManager.getProperties(c, eventData.getObjectURI());  // TODO: What is this?
```

Comment:
Can this TODO be deleted or addressed?

Reply:
I don't think it should be doing anything

Reply:
Ah yes, I forgot to get rid of that.  Removed and all tested ok.